locations/spiders/ollies_bargain_outlet.py

```diff
@@ -1,94 +1,43 @@
 import scrapy
 
+from locations.dict_parser import DictParser
 from locations.hours import OpeningHours
-from locations.items import GeojsonPointItem
 
 
 class OlliesBargainOutletSpider(scrapy.Spider):
     name = "ollies_bargain_outlet"
     allowed_domains = ["ollies.us"]
     item_attributes = {"brand": "Ollie's Bargain Outlet", "brand_wikidata": "Q7088304"}
+    custom_settings = {"ROBOTSTXT_OBEY": False}
 
     def start_requests(self):
         url = "https://www.ollies.us/admin/locations/ajax.aspx"
+        payload = "Page=0&PageSize=1&StartIndex=0&EndIndex=5&Longitude=-74.006065&Latitude=40.712792&City=&State=&F=GetNearestLocations&RangeInMiles=5000"
+        headers = {"content-type": "application/x-www-form-urlencoded; charset=UTF-8"}
 
-        headers = {
-            "origin": "https://www.ollies.us",
-            "Referer": "https://www.ollies.us/locations/",
-        }
+        yield scrapy.Request(url=url, method="POST", headers=headers, body=payload, callback=self.get_all_locations)
```

Comment:
If you do this as a `FormRequest` it'll do the headers for you, and encode the payload.

```suggestion
        yield FormRequest(
            url="https://www.ollies.us/admin/locations/ajax.aspx",
            formdata={"Page": "0",
            "PageSize": "1".
            ... },
            callback=self.get_all_locations,
        )
```